tpm2/tpm2.go

```diff
@@ -1211,6 +1211,32 @@ func NVDefineSpace(rw io.ReadWriter, owner, handle tpmutil.Handle, ownerAuth, au
 	return err
 }
```

Comment:
Do you think you could eliminate encodeDefineSpace by implementing NVDefineSpace on top of NVDefineSpaceNew?